src/twisted/conch/ssh/keys.py

```diff
@@ -958,8 +1018,13 @@ def type(self):
                 self._keyObject, (dsa.DSAPublicKey, dsa.DSAPrivateKey)):
             return 'DSA'
         elif isinstance(
-                self._keyObject, (ec.EllipticCurvePublicKey, ec.EllipticCurvePrivateKey)):
+                self._keyObject, (ec.EllipticCurvePublicKey,
+                                  ec.EllipticCurvePrivateKey)):
             return 'EC'
+        elif isinstance(
+                self._keyObject, (nacl.signing.VerifyKey,
+                                  nacl.signing.SigningKey)):
+            return 'ED25519'
```

Comment:
I think it's better to leave this all caps because the rest of the return strings are in all caps.